kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/coordinatedcommits/CommitCoordinatorClientSuite.scala

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (2024) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.kernel.defaults.internal.coordinatedcommits
+
+import io.delta.kernel.defaults.DeltaTableWriteSuiteBase
+import io.delta.kernel.internal.actions.Metadata
+import io.delta.kernel.internal.TableConfig
+import io.delta.storage.commit.{Commit, CommitCoordinatorClient, CommitResponse, GetCommitsResponse, UpdatedActions}
+import io.delta.storage.LogStore
+import io.delta.storage.commit.actions.{AbstractMetadata, AbstractProtocol}
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileStatus, Path}
+
+import java.{lang, util}
+import java.util.Collections
+import scala.collection.convert.ImplicitConversions.`map AsScala`
+import scala.collection.JavaConverters._
+
+class CommitCoordinatorClientSuite extends DeltaTableWriteSuiteBase {
```

Comment:
These are the tests for `CommitCoordinatorProvider`. Please rename to `CommitCoordinatorProviderSuite.scala`.